### PR TITLE
Sole trader enrolment

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/TT-1041)

sole trader

### CODE REVIEW

File: enrolment/templates/enrolment/companies-house-personal-details.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    </div>
+    <br>
+    <div class="grey-information-box">
+        <h3 class="heading-medium">Your company details</h3>
+        <p>{{ company.company_name }}</p>
+        <p>{{ company.address }}</p>
+        <a href="{% url 'enrolment-companies-house' step='business-details' %}" class="link">Change</a>
+    </div>
+    <div id="form-step-body-text">
+        <h1 class="heading-large">Enter personal details</h1>
```
Comment (eadpearce) [inline enrolment/templates/enrolment/companies-house-personal-details.html]: this `h1` is at the bottom. is this intentional? also, should the `h3`s above be `h2`s instead?

File: enrolment/templates/enrolment/companies-house-personal-details.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    </div>
+{% endblock %}
+
+{% block below_form %}
+    <p><a class="link" href="#">View the list of background checks here</a></p>
```
Comment (eadpearce) [inline enrolment/templates/enrolment/companies-house-personal-details.html]: placeholder link?

File: enrolment/templates/enrolment/sole-trader-business-details.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
+{% extends 'enrolment/wizard.html' %}
+
+{% load static from static %}
+
+{% block above_form %}
+    <div class="grey-information-box">
+        <h3 class="heading-medium">Type of business or organisation</h3>
```
Comment (eadpearce) [inline enrolment/templates/enrolment/sole-trader-business-details.html]: again, should this be an `h2` instead?

File: enrolment/templates/enrolment/sole-trader-personal-details.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{% extends 'enrolment/wizard.html' %}
+
+{% load static from static %}
+
+{% block above_form %}
+    <div class="grey-information-box">
+        <h3 class="heading-medium">Type of business or organisation</h3>
```
Comment (eadpearce) [inline enrolment/templates/enrolment/sole-trader-personal-details.html]: `h2`s?

File: enrolment/templates/enrolment/sole-trader-search.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
+{% extends 'enrolment/wizard.html' %}
+
+{% load static from static %}
+
+{% block above_form %}
+    <div class="grey-information-box">
+        <h3 class="heading-medium">Type of business or organisation</h3>
```
Comment (eadpearce) [inline enrolment/templates/enrolment/sole-trader-search.html]: h2? same as before

File: enrolment/templates/enrolment/companies-house-personal-details.html (hunk 2 above)
Comment (richtier) [reply enrolment/templates/enrolment/companies-house-personal-details.html]: Si. This is behind feature flag, and the is a ticket so this won't get lost :)

File: enrolment/templates/enrolment/companies-house-personal-details.html (hunk 1 above)
Comment (richtier) [reply enrolment/templates/enrolment/companies-house-personal-details.html]: image

The h1 is powering the `enter personal details`. That seems an appropriate h1, no?

Comment (eadpearce) [reply enrolment/templates/enrolment/companies-house-personal-details.html]: oh i see. fair

**Pull request merged**